static/js/components/VideoPlayer_test.js

```diff
@@ -304,4 +303,30 @@ describe("VideoPlayer", () => {
       }
     })
   })
+  ;[false, true].forEach(function(useYoutube) {
+    it(`checkYouTube ${expect(
+      useYoutube
+    )} be called if useYoutube is ${String(useYoutube)}`, async () => {
+      const wrapper = renderPlayer({ useYouTube: useYoutube })
```

Comment:
small nit: when you're sticking a variable into an object and you want it in the object under the same key as the variable's name there is a little syntax sugar:

```js
const foo = "hey!"
const fooInObj = { foo }
```

then `fooInObj.foo` will have the same value as `foo`.

Reply:
Do you mean instead of
```javascript
const wrapper = renderPlayer({ useYouTube: booleanTube })
```
do:
```javascript
const useYouTube = booleanTube
const wrapper = renderPlayer({ useYouTube})
```

Reply:
right now you have a variable `useYoutube`, if you just rename that variable to `useYouTube` you can use that syntax sugar. It's not a major thing, just wanted to point it out in case you weren't aware :)

Reply:
Got it, thanks